### PR TITLE
Document audio/video track usage

### PR DESCRIPTION
**Pull request by BrandonOCasey**
## Description
Document the recently added Video/Audio Track functionality

## Specific Changes proposed
* add the documentation to the guides folder
* move tracks guide to text-tracks
* changed index.md entry for tracks.md to make sense
* updated tracks.md to point to text-tracks, video-tracks, and audio-tracks

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [x] Reviewed by Two Core Contributors

### CODE REVIEW

File: docs/guides/audio-tracks.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Audio Tracks
+
+Audio Tracks are a function of HTML5 video for providing time triggered text to the viewer. Video.js makes audio tracks work across all browsers. There are currently five types of tracks:
```
Comment (nickygerritsen) [inline docs/guides/audio-tracks.md]: `triggered text`? Guess you mean something else?

File: docs/guides/text-tracks.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
+# Tracks
+
+Text Tracks are a function of HTML5 video for providing time triggered text to the viewer. Video.js makes tracks work across all browsers. There are currently five types of tracks:
```
Comment (nickygerritsen) [inline docs/guides/text-tracks.md]: Here as well `triggered text`.

File: docs/guides/video-tracks.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# Video Tracks
+
+Video Tracks are a function of HTML5 video for providing time triggered text to the viewer. Video.js makes video tracks work across all browsers. There are currently six types of tracks:
```
Comment (nickygerritsen) [inline docs/guides/video-tracks.md]: Here as well `triggered text`?

**Comment by nickygerritsen** [comment]
Sorry typed one comment at a wrong place but deleted it again :)

File: docs/guides/video-tracks.md (hunk 3 above)
Comment (BrandonOCasey) [inline docs/guides/video-tracks.md]: good catch on these, copy pasted the format of text-track but never updated this section

**Comment by nickygerritsen** [comment]
OK LGTM now :)

**Comment by BrandonOCasey** [comment]
well this was a quick one! thanks!

**Comment by gkatsev** [comment]
LGTM

**Pull request closed without merging**